index.html

```diff
@@ -1166,14 +1166,17 @@ <h3>Contexts</h3>
         <h3>Identifiers</h3>
 
         <p>
-When expressing statements about a specific thing, such as a person, product,
-or organization, it is often useful to use some kind of identifier so that
-others can express statements about the same thing. This specification defines
-the optional <code>id</code> <a>property</a> for such identifiers. The
-<code>id</code> <a>property</a> is intended to unambiguously refer to an object,
-such as a person, product, or organization. Using the <code>id</code>
-<a>property</a> allows for the expression of statements about specific things in
-the <a>verifiable credential</a>.
+When expressing statements about a specific thing, such as a person, product, or
+organization, it is often useful to use a globally unique identifier for it.
+Globally unique identifiers are useful so that others can express statements
+about the same thing. This specification defines the optional <code>id</code>
+<a>property</a> for such identifiers. Using the <code>id</code> <a>property</a>
+allows for the expression of statements about specific things in the
+<a>verifiable credential</a> and is set by an <a>issuer</a> when creating
+objects in a <a>verifiable credential</a> or a <a>holder</a> when creating
+objects in a <a>verifiable presentation</a>. Examples of <code>id</code>s
+include UUIDs (`urn:uuid:0c07c1ce-57cb-41af-bef2-1b932b986873`), HTTP URLs
+(`https://id.example/123`), and DIDs (`did:example:1234abcd`).
```

Comment:
While the current text in the pull request suggests using HTTP URLs as identifiers (e.g., `https://id.example/123`), I would like to argue that using hash URIs with a fragment identifier is a more appropriate approach for identifying a person, place, or thing.

The W3C has addressed this topic in a TAG resolution called HR14 (Hash In URI). The resolution suggests that HTTP URIs without fragment identifiers should be used for documents, while hash URIs with fragment identifiers should be used for identifying resources other than documents, such as people, places, or things. You can find more information about this resolution in the following W3C document: https://www.w3.org/2001/tag/doc/hash-in-uri

The main rationale behind using hash URIs with fragment identifiers is that it allows for better separation between the document that describes the resource and the resource itself. This approach helps in avoiding confusion and allows for more precise statements about the resource.

Considering this, I suggest updating the example in the pull request to include a hash URI with a fragment identifier for identifying a person, place, or thing, such as `https://example.com/people#alice`. This way, the identifier clearly points to a non-document resource, in line with the W3C HR14 resolution.

Here's a revised version of the text:

Globally unique identifiers are useful so that others can express statements about the same thing. This specification defines the optional `id` property for such identifiers. Using the `id` property allows for the expression of statements about specific things in the verifiable credential and is set by an issuer when creating objects in a verifiable credential or a holder when creating objects in a verifiable presentation. Examples of `id`s include UUIDs (`urn:uuid:0c07c1ce-57cb-41af-bef2-1b932b986873`), HTTP URLs with fragment identifiers (`https://example.com/people#alice`), and DIDs (`did:example:1234abcd`).

Reply:
@melvincarvalho — I am somewhat concerned that [the referenced W3C document](https://www.w3.org/2001/tag/doc/hash-in-uri) has internal links to ["This version"](http://www.w3.org/TR/2009/WD-hash-in-url-20090415/) and ["Latest Version"](http://www.w3.org/TR/hash-in-url/), both in `www.w3.org/TR/`, but both `404`. I also see that the _Status of this Document_ is that it was "published as a Public Working Draft" in 2009 and hasn't been updated since. This significantly lowers my inclination to follow its guidance.

Reply:
Thank you, @TallTed, for your feedback. I understand your reservations given the accessibility issues with the referenced W3C document and its dated status. The 404 links have now been fixed.  I believe the essence of the TAG resolution encapsulates remains relevant, particularly in the context of our work here.

The resolution outlines a distinction between identifiers for documents versus non-document resources, a principle that I believe could contribute to the clarity and accuracy of our identifiers. While the document itself may no longer be maintained, the principles it outlines remain sound and adopted in various contemporary practices.

That said, I would encourage aligning with the TAG finding, particularly the use of hash URIs with fragment identifiers for non-document resources, despite the age of the document. After all, the longevity of these principles attests to their utility and relevance.

Reply:
@melvincarvalho — Please note that while the linked document now has different (working) "This version" and "Latest Version" targets, neither is accurate. Note that the Abstract says, "This draft finding is being prepared in response to [TAG issue #60](http://www.w3.org/2001/tag/group/track/issues/60)." (That would certainly have been better noted in the SotD.)

Going to that TAG issue, I find that while it links to the above document, it also says that (emphasis mine), "_Current_ work is: [Repurposing the Hash Sign for the New Web; W3C Working Draft: Putative TAG Finding 28 February 2011](http://www.w3.org/2001/tag/2011/02/HashInURI-20110228)" which is two years more recent, but still outdated.

The first exterior link found in both as an example, in [Addressing Into Multimedia Streams -- CNN](https://www.w3.org/2001/tag/2011/02/HashInURI-20110228.html#MultimediaStreamsCNN), is `http://www.cnn.com/video/#/video/tech/2008/02/19/vo.aus.sea.spider.ap` which was probably accurate in 2011, but it is not today, as that link gets redirected to the fragmentless `https://www.cnn.com/videos/tech/2008/02/19/vo.aus.sea.spider.ap` (which is also now a 404).

I submit that the DRAFT TAG finding should be more carefully considered and potentially updated, given the changes in the wild.

Reply:
@TallTed , your comments seem to divert from the core issue at hand. It's crucial to focus on the present topic and not get lost in the minutiae of details from documents. I respectfully suggest that you update your understanding with the reading materials before making assertions. This will ensure we're all on the same page, and that our discussions are productive and relevant. Let's work together to address the important issues at hand rather than getting sidetracked.